metricbeat/module/docker/cpu/helper.go

```diff
@@ -80,27 +80,30 @@ func getNewCpu(stats *dc.Stats) CPURaw {
 
 func (c *CPUService) PerCpuUsage(stats *dc.Stats) common.MapStr {
 	var output common.MapStr
-	if cap(getNewCpu(stats).PerCpuUsage) == cap(getOLdCpu(stats).PerCpuUsage) {
+	if cap(getNewCpu(stats).PerCpuUsage) == cap(getOldCpu(stats).PerCpuUsage) {
```

Comment:
Ok. Now i'm convinced :) Thanks for the explainations 👍 👍 
I'll change it if you don't mind !